<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVShadingCalculationDialog</class>
 <widget class="QDialog" name="SVShadingCalculationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxSimulationTimeFrame">
     <property name="title">
      <string>Simulation time and duration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labelStart">
        <property name="text">
         <string>Start date:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditStartDate">
        <property name="placeholderText">
         <string>dd.mm.yyyy hh:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Duration:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QtExt::ParameterEdit" name="lineEditDuration"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxUnitDuration"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelEnd">
        <property name="text">
         <string>End date:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditEndDate">
        <property name="placeholderText">
         <string>dd.mm.yyyy hh:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Simulation Parameter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Grid Size [m]:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QtExt::ValidatingLineEdit" name="lineEditGridSize"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Sun Mapping [Deg]:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QtExt::ValidatingLineEdit" name="lineEditSunCone"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>200</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCalculate">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonChancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QtExt::ParameterEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ParameterEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonChancel</sender>
   <signal>clicked()</signal>
   <receiver>SVShadingCalculationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
